### PR TITLE
Util main updates

### LINKED ISSUE #57: Implement shuffling on the playlist level
add shuffling to playlists if required

### PR DESCRIPTION
**Pull request by KittenMoon**
Added getters and setters for most values in channels, playlists and images

### CODE REVIEW

**Comment by KittenMoon** [comment]
this will now have shuffle and flatten, and picking the current channels too

File: discus/util/db.py [hunk 1]
```diff
@@ -2,34 +2,43 @@
...
+      if (attempt == 0):
+        print("Attempting to connnect to the database...")
+      else:
+        print("Reattempting to connect to the database...")
+      client = pymongo.MongoClient("mongodb+srv://" + db_username + ":" \
+        + db_password + "@" + db_host + "/")
+      
+      # Check if connected.
+      client.server_info()
+      print("connected!")
```
Comment (cosmicboots) [inline discus/util/db.py]: If it connects successfully, we should break out of the loop to keep it from trying to do it again.

Comment (KittenMoon) [reply discus/util/db.py]: yeah, that is planned to be added, just not pushed yet

**Review by cosmicboots — APPROVED** [review]
LGTM

**Pull request merged**